config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -52,9 +52,6 @@ periodics:
           echo "BAZEL_VERSION - Falling back to 5.3.0";
         fi;
         /workspace/test-infra/images/kubekins-e2e/install-bazel.sh;
-        go install sigs.k8s.io/kubetest2@latest;
```

Comment:
there should be at least one job somewhere that is testing @ latest, the first kubetest2 jobs were setup to provide signal for kubetest2's own development

Reply:
there is, the long list of presubmits here 

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kubetest2/kubetest2-canaries.yaml